plugin/src/main/groovy/com/novoda/staticanalysis/internal/findbugs/FindbugsConfigurator.groovy

```diff
@@ -62,12 +62,16 @@ class FindbugsConfigurator extends CodeQualityConfigurator<FindBugs, FindBugsExt
             sourceFilter.applyTo(task)
             task.conventionMapping.map("classes", {
                 List<String> includes = createIncludePatterns(task.source, androidSourceDirs)
-                project.fileTree(variant.javaCompile.destinationDir).include(includes)
-            }.memoize());
```

Comment:
During the investigation of the bug I realised that the `classes` property is evaluated multiple times during the configuration phase of an Android project, so I decided to avoid possible side-effects of memoization.